docs/architecture.md

```diff
@@ -191,23 +191,31 @@ so their source IP will be rewritten to the Node's IP before going out.
 
 ### ClusterIP Service
 
-<img src="/docs/assets/service_walk.svg.png" width="600" alt="Antrea Service Traffic Walk">
+Antrea supports two ways to implement Services of type ClusterIP - leveraging
+`kube-proxy`; or AntreaProxy that implements load balancing for ClusterIP
+Service traffic with OVS.
+
+In the first way of leveraging `kube-proxy`, Antrea Agent adds OVS flows to
```

Comment:
s/In the first way of leveraging `kube-proxy`/When leveraging `kube-proxy`

Reply:
Done.